src/card.css

```diff
@@ -0,0 +1,110 @@
+
+.card {
+  width: 198px;
+  background-color: var(--backgroundColor);
+  box-sizing: content-box;
+  border: 1px solid var(--borderColor);
+  border-radius: 5px;
+  font-family: "Roboto", sans-serif;
+}
+
+.card__conteiner {
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+}
```

Comment:
Class .card__conteiner is useless, because u have .card class, and it's a block. U can set in .card box-sizing: border-box and set paddings there.